sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/README.md

```diff
@@ -1,377 +0,0 @@
-# Azure Storage Queue client library for Java
```

Comment:
was deleting this README intentional with this change to add a CHANGELOG?

Reply:
Not really. I just deleted the duplicate one. I can open another PR for the deletion. Thanks!